test/SILOptimizer/access_enforcement_selection.sil

```diff
@@ -99,7 +99,7 @@ bb0:
   return %9 : $()
 }
 
-sil [dynamically_replacable] [ossa] @closureCapturingByStorageAddress2 : $@convention(thin) (@inout_aliasable Builtin.Int64) -> ()
+sil [dynamically_replaceable] [ossa] @closureCapturingByStorageAddress2 : $@convention(thin) (@inout_aliasable Builtin.Int64) -> ()
```

Comment:
That needs to be fixed in SILPrinter + SILParser first

Reply:
I think the best is if you create a separate PR just fixing dynamically_replacable.